File: src/main/java/com/ciazhar/authserver/repository/PermissionRepository.java
package com.ciazhar.authserver.repository;

import com.ciazhar.authserver.model.Permission;
import org.springframework.data.repository.PagingAndSortingRepository;

/**
 * Created by ciazhar on 6/23/17.
 */

public interface PermissionRepository extends PagingAndSortingRepository<Permission,String> {

}

File: src/main/kotlin/com/ciazhar/authserver/repository/PermissionRepository.kt
package com.ciazhar.authserver.repository

import com.ciazhar.authserver.model.jpa.Permission
import org.springframework.data.repository.PagingAndSortingRepository

/**
 * Created by ciazhar on 6/23/17.
 */

interface PermissionRepository : PagingAndSortingRepository<Permission, String>